release_notes/ocp_3_4_release_notes.adoc

```diff
@@ -119,43 +490,276 @@ Downgrading from Elasticsearch 2.4 to Elasticsearch 1.x is not possible due to
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1367379[*BZ#1367379*])
+
+* The web console would not show any errors on the *Overview* page when metrics were configured, but not working. It would quietly fall back to the behavior when metrics were not set up. The web console now shows an error message with a link to the metrics status URL to help diagnose problems such as invalid certificates. The alert can be permanently dismissed for users who do not want to see it.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1382728[*BZ#1382728*])
+
+* In some cases, the Y-axis values would not adjust to fit the data when looking at metrics for a pod. The Y-axis now scales appropriately to fit the data as usage increases.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1386708[*BZ#1386708*])
+
+* If you deleted a pod and created a new pod with the same name, you would see metrics for the previous pod when viewing metrics. Only metrics for the new pod are now shown.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1386838*[BZ#1386838*])
+
+* When a pod had more than one container, the web console was incorrectly showing total memory and CPU usage for all containers in the pod on the metrics page rather than only the selected container. This could make it appear that memory usage exceeded the limit set for the container. The web console now correctly shows the memory and CPU usage only for the selected container.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1387274[*BZ#1387274*])
+
+* The logo and documentation links must be changed for each release. This was not yet completed, so the logo and documentation links represented OpenShift Origin instead of OpenShift Container Platform. The appropriate logo and links for the release were added and are now correct.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1388798[*BZ#1388798*])
+
+* Previously, you could select  *Push Secret* and *Pull Secret* on the DC editor page and on the *Create From Image* page. These options are not helpful on these pages because they are using integrated registry. Therefore, the *Push Secret* and *Pull Secret* select boxes are now removed from the DC editor and *Create From Image* pages and users can no longer select these options.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1388884[*BZ#1388884*])
+
+* Routes popover warning messages were being truncated at the end of the string. Before the relevant portion of the warning message could be displayed, the certificate content results in the warning message were being truncated. After the bug fix, the truncation of the warning message was changed from truncating at the end of the string to truncating in the middle of the string. As a result, the relevant portion of the warning message is now visible. (BZ#1389658)
```

Comment:
Yeah, this chunk got added last and I didn't make em fancy links yet. Doing so now.